src/assets/css/books.css

```diff
@@ -26,32 +56,68 @@
   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
 }
 
-.book-card {
-  width: 33%;
-  height: 20%;
-}
-
 .add-book-form,
 .book-card {
   width: 100%;
   height: 33%;
   display: flex;
+  background-color: white;
+  margin: 1% auto 0 auto;
+  padding: 0 0 3% 3%;
+}
+
+.book {
+  width: 50%;
+}
+
+.text {
+  box-sizing: content-box;
+  padding: 0;
+  margin-top: 5%;
+}
+
+.text h2,
+.text p {
+  padding: 0;
+  margin: 0;
+}
+
+.buttons {
+  margin-top: 5%;
+}
 
-  /* flex-direction: column; */
+.progress {
+  width: 25%;
+}
+
+.current-chapter {
+  width: 25%;
 }
 
 .add-book-form {
+  top: 10%;
   flex-direction: column;
+  background-color: var(--my-lightGray);
 }
 
 form {
   width: 100%;
+  display: flex;
 }
 
-label {
-  width: 100%;
+/* input {
+  width: 90%;
+} */
+
```

Comment:
### Optional
- [ ] It would be nice if we remove all commented piece of code, it makes our codebase looks messy and difficult to understand